src/modules/delete.js

```diff
@@ -0,0 +1,22 @@
+const task = document.querySelectorAll('.task');
+
+const todos = JSON.parse(localStorage.getItem('todo-list')) || [];
+
+const deleteTask = () => {
+  task.forEach((t) => {
```

Comment:
Great work using es6 modules but I would suggest you use a clear parameter name that is meaningful so others who might work with you in the future will understand the purpose of your code.

Reply:
Thank you @shakerAbuDrais For reviewing my code :))